src/region_growing/abstract.py

```diff
@@ -0,0 +1,39 @@
+"""Data Fuser Abstract Base Class"""
```

Comment:
Waarom is deze class nodig als het gewoon een kopie is van de AbstractFuser?

Reply:
Ah ik zie het, je hebt de method signature aangepast. Misschien nog even de doctrings bijwerken :)